### PR TITLE
Added accelerator CRD

### PR DESCRIPTION
**Pull request by Gkrumbach07**
closes: https://github.com/opendatahub-io/odh-dashboard/issues/1371

## Description

add a CRD to the manifest folder to represent accelerator devices 

## How Has This Been Tested?

- apply this yaml in opendatahub

## Test Impact

n/a backend

## Request review criteria:

- [x] The commits have meaningful messages (squashes happen on merge by the bot).
- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work
- [x] The developer has added tests or explained why testing cannot be added (unit tests & storybook for related changes)

### CODE REVIEW

**Review by lucferbux — APPROVED** [review]
/lgtm

File: manifests/crd/acceleratorprofile.opendatahub.io.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1
```
Comment (lucferbux) [inline manifests/crd/acceleratorprofile.opendatahub.io.crd.yaml:1]: Well, after approving it... I realize, do you want only to have the spec or do you want the operator to apply it?
If so, you need to add it into the `kustomize` file, I'll wait for your confirmation @Gkrumbach07 if that's the case

Comment (Gkrumbach07) [reply manifests/crd/acceleratorprofile.opendatahub.io.crd.yaml:1]: operator is going to need to apply it

Comment (lucferbux) [reply manifests/crd/acceleratorprofile.opendatahub.io.crd.yaml]: Great, so last changes are fine now!

**Review by lucferbux — APPROVED** [review]
/lgtm

**Comment by andrewballantyne** [comment]
After the fact review -- Looks good, modified date probably still needs to be thought on if it needs to be a property or not. We'll also need to look into tolerations and how that'll work too. 

Likely a read through of https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/ to figure out what we need to make it happen.

**Pull request merged**